pkg/codegen/docs/gen.go

```diff
@@ -418,6 +418,9 @@ func (mod *modContext) typeString(t schema.Type, lang string, characteristics pr
 		tokenName := tokenToName(t.Token)
 		// Links to anchor tags on the same page must be lower-cased.
 		href = "#" + strings.ToLower(tokenName)
+	default:
+		// Assume primitive/built-in type if no match for cases listed above.
+		href = docLanguageHelper.GetDocLinkForBuiltInType(t.String())
```

Comment:
Actually I don't think we can assume that it would be a primitive type. The schema has other types besides just Array and Object types.

I think what we should do instead is to check if the type is in-fact a primitive type by calling `schema.IsPrimitiveType(t)` and then calling `GetDocLinkForBuiltInType` to get try and get the link for it.